ocrd_tesserocr/recognize.py

```diff
@@ -194,8 +218,10 @@ def _process_lines(self, tessapi, textlines, region_image, region_xywh):
                 line, region_image, region_xywh)
             # todo: Tesseract works better if the line images have a 5px margin everywhere
             tessapi.SetImage(line_image)
-            # RAW_LINE fails with pre-LSTM models, but sometimes better with LSTM models
-            tessapi.SetPageSegMode(PSM.SINGLE_LINE)
+            if self.parameter['raw_lines']:
```

Comment:
Will there be a need for another PSM in the future? If so, it might be more intuitive to just follow Tesseracts nomenclature and pass a parameter `PSM` on to Tesseract.

Reply:
> If so, it might be more intuitive to just follow Tesseracts nomenclature and pass a parameter `PSM` on to Tesseract.

...instead of `textequiv_level` you mean? The problem is that Tesseract's PSMs are already non-orthogonal and more fine-grained than our/PAGE's hierarchy. Plus we should strive to encapsulate Tesseract's idiosyncrasies here. 

This new distinction is necessary though, because for LSTM models, PSM.SINGLE_LINE can be detrimental, esp. on images that have already been masked from polygonal line segmentation. Moreover, with current Tesseract, this can still crash under rare circumstances. So this depends on models and data, and we should at least provide the old PSM as an opt-out.

Reply:
> ...instead of `textequiv_level` you mean?

Instead of `raw_lines`. But you're right, this is about `RAW_LINE and `SINGLE_LINE` and the parameter is perfectly expressive for that.